config/utils.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = {
+  isPresent: variable => Boolean(variable),
   asBoolean: variable => ['true', '1'].includes(variable),
```

Comment:
Would have this solved the problem too?

```suggestion
  asBoolean: variable => [true, 1, 'true', '1'].includes(variable),
```

Reply:
No, the DSN is a string, we just want to check if it’s present or not to activate Sentry. Also from what I understand, environment variables are always string so it’s not possible to have `true` or `1`, right?

Reply:
You’re right, I didn’t fully understand the problem 😄